src/vector/layer.rs

```diff
@@ -13,6 +13,73 @@ use std::{convert::TryInto, ffi::CString, marker::PhantomData};
 
 use crate::errors::*;
 
+/// Layer capabilities
+pub enum LayerCaps {
+    /// Layer capability for random read
+    OLCRandomRead,
```

Comment:
```suggestion
    RandomRead,
```

Just a style nit, but since this is already namespaced under `Layer::LayerCaps`, do we get anything with the uniformly repeated `OLC` prefix for each case?

Reply:
Oh, the idea was to simply to stick to the Gdal original naming.

Reply:
:+1: for dropping the prefix, although we don't really have prior art for this.

Reply:
I'm really not sure with that: this would be inconsistent with other refererences to gdal constants like `OGRwkbGeometryType` or other type imported from `gdal-sys`. Imho, constant naming should stick to Gdal name or be renamed for all constant definition, but not a mix of both naming convention.